csrc/scheduler/normalization_inner_outer.cpp

```diff
@@ -141,10 +146,6 @@ std::unique_ptr<ReductionParams> getInnerOuterPersistentHeuristics(
         runtime_info.getIndexType());
   }
 
-  // save persistent tvs should use shared memory, to avoid calling
-  // getPersistentBufferStorageParams again during the scheduling.
-  rparams->smem_persistent_buffers = buffer_params.smem_persistent_buffers;
-
```

Comment:
move to front, so `getHeuristics()` can print the correct `smem_persistent_buffers`.